include/RED4ext/Api/EMainReason.hpp

```diff
@@ -10,6 +10,7 @@ namespace RED4ext
 enum class EMainReason : uint8_t
 {
     Load = 0,
-    Unload
+    Unload,
+    Run
```

Comment:
What would be the advantage of having this instead of just using GameStates?

Reply:
What about calling it on `Running::OnTick` and change it to `Tick`?

Reply:
The only advantage is easier entry for beginners. I didn't suggest adding it though, just gave an example of how it could be handled. In `Load` you can't even do anything engine/game related. The game states are hard to understand and at least need a better description, explaining which systems exactly are initialized at what point.

Reply:
I'm fine with keeping it.